docs/examples.md

```diff
@@ -80,6 +89,18 @@ Build and run WildFly applications on CentOS 7. For more information about using
 
 ## Binary example
 
+### java 
+
+Java can be used to deploy binary artifact
```

Comment:
`Java may be used to deploy a binary artifact, for example:`

Or something similar ^^